Source/multi.cpp

```diff
@@ -415,6 +416,8 @@ void UnregisterNetEventHandlers()
 
 bool InitSingle(GameData *gameData)
 {
+	Players.emplace_back();

```

Comment:
Should this be Players.resize(1); to avoid issues with it no longer being cleared during demo playback?

Reply:
Yeah I guess so